azuread/config.go

```diff
@@ -8,7 +8,9 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Azure/azure-sdk-for-go/services/domainservices/mgmt/2017-06-01/aad"
 	"github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-09-01/network"
```

Comment:
We shouldn't be doing anything with the network API in this provider.

Reply:
But creating aadds needs virtual network as input, and after creating succeeds, we should modify virtual network dnsserver. I wanna Is there any other ways to handle matters like this ?
otherwise in tf files, users need to apply first, then update virtual network dnsserver mannually and apply again, it's convenient to do automation